### PR TITLE
Evaluation theory

### PR DESCRIPTION
**Pull request by TheRumle**
Reading guide: 
The focus for this PR has been to find and use theory about how one uses a prototype to evaluate and create usability test. We evaluate two approaches to usability testing, and pick one suitable for us - in this case the approach must be suitable for examining whether the product (prototype) would solve the issues presented in the initial interviews. 

After establishing which approach to usability testing to use, a test plan has been created - this test plan follows state-of-the-art approach. 

We explain how we implement this test plan, and describe the overall approach for the testing. 

The next thing in the HCI saga is to describe the results of the test.

* Created appendices for usability test

### CODE REVIEW

File: report/src/_preamble-blx.bib
Comment (chhoumann) [inline report/src/_preamble-blx.bib:1]: Don't commit this

File: report/src/bib/mybib.bib [hunk 1]
```diff
@@ -532,3 +532,112 @@ @article{gruberCO2SensorsOccupancy2014
...
+  publisher={CRC Press}
+}
+
+@CONFERENCE{NielsenLandauer1993,
+author = {Nielsen, Jakob and Landauer, Thomas K.},
+title = {Mathematical model of the finding of usability problems},
+year = {1993},
+journal = {Conference on Human Factors in Computing Systems - Proceedings},
+pages = {206-213},
+url = {https://www.scopus.com/inward/record.uri?eid=2-s2.0-0027850828&partnerID=40&md5=482b7aaf88c082b6d0a283035e0bce99%7D,
```
Comment (chhoumann) [inline report/src/bib/mybib.bib:611]:
```suggestion
url = {https://www.scopus.com/inward/record.uri?eid=2-s2.0-0027850828&partnerID=40&md5=482b7aaf88c082b6d0a283035e0bce99%7D},
```

File: report/src/bib/mybib.bib [hunk 2]
```diff
@@ -532,3 +532,112 @@ @article{gruberCO2SensorsOccupancy2014
...
+publisher = {Association for Computing Machinery}, 
+address = {New York, NY, USA}, 
+volume = {53}, number = {5}, 
+issn = {0001-0782}, 
+url = {https://doi.org/10.1145/1735223.1735255%7D, 
+doi = {10.1145/1735223.1735255},
+journal = {Commun. ACM}, month = {may}, 
+pages = {130-133}, 
+numpages = {4} 
+}
```
Comment (chhoumann) [inline report/src/bib/mybib.bib:642]:
```suggestion
@article{Hwang_PeopleRequiredForUsabilityEvaluation, 
author = {Hwang, Wonil and Salvendy, Gavriel}, 
title = {Number of People Required for Usability Evaluation: The 10±2 Rule}, 
year = {2010}, 
issue_date = {May 2010}, 
publisher = {Association for Computing Machinery}, 
address = {New York, NY, USA}, 
volume = {53}, 
number = {5}, 
issn = {0001-0782}, 
url = {https://doi.org/10.1145/1735223.1735255%7D}, 
doi = {10.1145/1735223.1735255},
journal = {Commun. ACM}, 
month = {may}, 
pages = {130-133}, 
numpages = {4} 
}
```

File: report/src/bib/mybib.bib
Comment (chhoumann) [inline report/src/bib/mybib.bib:1]: Please start checking that you aren't breaking the Bib file...

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 3]
```diff
@@ -1,3 +1,71 @@
-\subsection{Evaluating prototype usability}
-This will be a future section about usability evaluation techniques.
-    
\ No newline at end of file
+\section{Design evaluation}\label{sec:prototype_evaluation}
+Having designed a prototype for a system meant to satisfy the stakeholders' requirements formed by user stories (see Appendix \ref{appendix:user_stories}), we can evaluate whether this system constitutes a suitable solution to the problematic situations described in Section \ref {sec:understanding_the_problem}.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:2]: Whitespace in ref
```suggestion
Having designed a prototype for a system meant to satisfy the stakeholders' requirements formed by user stories (see Appendix \ref{appendix:user_stories}), we can evaluate whether this system constitutes a suitable solution to the problematic situations described in Section \ref{sec:understanding_the_problem}.
```

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 4]
```diff
@@ -1,3 +1,71 @@
...
+Furthermore, we want to use the prototype to evaluate whether the system has a high degree of usability. 
+We do this to understand whether the system design is suitable for the stakeholders.
+
+When evaluating a prototype, multiple approaches can be taken.
+This section covers two types of usability testing and their strengths and shortcomings.
+When choosing which type of usability test to use, we consider its strengths in terms of whether resources for the test approach can be realistically allocated by all parties and whether the approach solves the problematic situations described in Section \ref {sec:understanding_the_problem}.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:8]: Whitespace in ref
```suggestion
When choosing which type of usability test to use, we consider its strengths in terms of whether resources for the test approach can be realistically allocated by all parties and whether the approach solves the problematic situations described in Section \ref{sec:understanding_the_problem}.
```

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+
+Many types of inspection methods can be used when performing expert-based usability testing.
+Consistency inspection involves establishing whether the design is consistent with existing designs \cite{NJ_InspectionMethods,wixon_consistencyInspections}.  
+Cognitive walkthroughs involve experts acting as if they were system users performing a series of tasks with the prototype \cite{Hollingsed_cognitive_walthrough}.
+
+A different approach to evaluation is user-based testing.
+User-based testing is often performed using a group of users representative of the overall population of users. \cite{researchmethodsinhumancomputerinteraction}.
+User-based testing is based on the idea that designers have preconceived opinions about how a system should be interacted with. 
+Therefore, the designers are likely to create designs that suit their view of how a design should be.However, this design does not necessarily align with how users imagine the system \cite{user-centred-design,researchmethodsinhumancomputerinteraction}.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:23]:
```suggestion
Therefore, the designers are likely to create designs that suit their view of how a design should be.
However, this design does not necessarily align with how users imagine the system \cite{user-centred-design,researchmethodsinhumancomputerinteraction}.
```

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 6]
```diff
@@ -1,3 +1,71 @@
...
+Two approaches can be used when conducting user-based testing. 
+Formative user-based testing is used early in the design phase to evaluate early designs. 
+The evaluation mainly focuses on finding design alternatives and often uses low-fidelity prototypes \cite{dumasAndFox_formative_usability_testing}.
+
+Contrary to formative testing, summative testing is performed late in the development of the design and often uses a hifi prototype.
+Here, the focus is on the evaluate whether the design satisfies specific design goals. \cite{dumasAndFox_formative_usability_testing}
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:29]: "the focus is on the evaluate"?

Not sure what that means

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 7]
```diff
@@ -1,3 +1,71 @@
...
+In our case, the design goals would constitute whether the design solves the problems described in Section \ref{sec:understanding_the_problem}..
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:30]:
```suggestion
In our case, the design goals would constitute whether the design solves the problems described in Section \ref{sec:understanding_the_problem}.
```

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 8]
```diff
@@ -1,3 +1,71 @@
...
+
+When choosing which type of usability test to use, we consider its strengths in terms of whether resources for the test approach can be realistically allocated by all parties and whether the approach solves the problematic situations.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:32]: Almost complete duplicate sentence of line 8 ??

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 9]
```diff
@@ -1,3 +1,71 @@
...
+The selected approach must be suitable for establishing the design's strength in terms of solving the problematic situation. 
+The testing approach should also be carried out in a manner such that it does not exceed the resources allocated by the stakeholders. 
+As expert-based solutions rely on the testers having not taken part in the designing of the system, and as it does not aim to find out how well the users of the system can solve specific tasks, we rule this approach out.
+
+On the other hand, a summative user-based testing approach specifically focuses on whether establishing whether the design satisfies specific design goals and whether these goals can be completed by a user. 
+Therefore, we select summative user-based testing as a preferred approach to establishing if the design of the developed hifi prototype constitutes a suitable solution to one or more of the problematic situations. 
+
+\subsection{Test plan}\label{sec:prototype_test_plan}
+When designing usability tests several things need to be considered.
+First and foremost, a procedure establishing who to participate in the test and how to perform the test must be selected.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:44]:
```suggestion
First and foremost, a procedure establishing who should participate in the test and how to perform the test must be selected.
```

File: report/src/problem_analysis/hci/prototype_evaluation.tex [hunk 10]
```diff
@@ -1,3 +1,71 @@
...
+\item Select representative users
+\item Select the setting
+\item Decide what tasks users should perform
+\item Decide what type of data to collect
+\item Before the test session (informed consent, etc.)
+\item During the test session
+\item Debriefing after the session
+\item Summarize results and suggest improvements
+\end{itemize}
+In the following section, we detail how we apply each step in \citeauthor{lazar2005web}s procedure.
```
Comment (chhoumann) [inline report/src/problem_analysis/hci/prototype_evaluation.tex:56]: Which section?

**Review by chhoumann — CHANGES_REQUESTED** [review]
Fine, but you have some sloppy errors

File: report/src/problem_analysis/hci/prototype_evaluation.tex (hunk 8 above)
Comment (TheRumle) [reply report/src/problem_analysis/hci/prototype_evaluation.tex:33]: Good catch, we yeet this line and keep line 8

**Review by chhoumann — APPROVED** [review]

**Pull request merged**